Description: Skip installing packages which currently don't work.
 Why don't they work:
 * Need csdp:
   - SemidefiniteProgramming
   - SumsOSquares
 * Need topcom:
   - Polyhedra
   - ToricInvariants

--- a/M2/Macaulay2/packages/=distributed-packages
+++ b/M2/Macaulay2/packages/=distributed-packages
@@ -41,7 +41,6 @@
 EdgeIdeals
 FourTiTwo
 StatePolytope
-Polyhedra
 Truncations
 Polymake
 gfanInterface
@@ -182,10 +181,7 @@
 NCAlgebra
 SpaceCurves
 ExteriorIdeals
-ToricInvariants
 SegreClasses
-SemidefiniteProgramming
-SumsOfSquares
 MultiGradedRationalMap
 VirtualResolutions
 Quasidegrees
